RiotSwiftUI/Modules/Common/Util/RoundedBorderTextField.swift

```diff
@@ -72,18 +83,33 @@ struct RoundedBorderTextField: View {
                         .foregroundColor(theme.colors.tertiaryContent)
                         .lineLimit(1)
                 }
-                ThemableTextField(placeholder: "", text: $text, configuration: configuration, onEditingChanged: { edit in
-                    self.editing = edit
-                    onEditingChanged?(edit)
-                })
-                .makeFirstResponder(isFirstResponder)
-                .onChange(of: text, perform: { newText in
-                    onTextChanged?(newText)
-                })
-                .frame(height: 30)
-                .modifier(ClearViewModifier(alignment: .center, text: $text))
+                if isEnabled {
+                    ThemableTextField(placeholder: "", text: $text, configuration: configuration, onEditingChanged: { edit in
+                        self.editing = edit
+                        onEditingChanged?(edit)
+                    })
+                    .makeFirstResponder(isFirstResponder)
+                    .onChange(of: text, perform: { newText in
+                        onTextChanged?(newText)
+                    })
+                    .frame(height: 30)
+                    .modifier(ClearViewModifier(alignment: .center, text: $text))
+                } else {
+                    ThemableTextField(placeholder: "", text: $text, configuration: configuration, onEditingChanged: { edit in
+                        self.editing = edit
+                        onEditingChanged?(edit)
+                    })
+                    .makeFirstResponder(isFirstResponder)
+                    .onChange(of: text, perform: { newText in
+                        onTextChanged?(newText)
+                    })
+                    .frame(height: 30)
+                    .allowsHitTesting(false)
+                    .opacity(0.5)
```

Comment:
Same comment.

Reply:
Not sure to follow you on this one as a text input can have some text set and be also disabled. I'm not sure we can avoid `if isEnabled` in this case.